test/src/index.js

```diff
@@ -29,4 +29,5 @@ import './core-layers';
 // TODO - Tests currently only work in browser
 if (typeof document !== 'undefined') {
   require('./experimental-layers');
+  require('./lite');
```

Comment:
This is broken under node when importing luma.gl (tries to access `document`). Need to fix luma.